docs/using-radicle/pushing-and-pulling-changes.md

```diff
@@ -23,44 +23,44 @@ It can be pushed to Radicle with:
 
 ### Following projects
 
-To contribute to another user's project to your machine, you must first find it on the Radicle network. To find a project, you need it's [**projectID**](understanding-radicle/glossary.md/#project-id). This can be found under in the project page under the [project name](understanding-radicle/glossary.md/#project-name), or in a seed node UI. 
+To contribute to another user's project, you must first find it on the Radicle network. To find a project, you need its [**projectID**](understanding-radicle/glossary.md/#project-id). This can be found in the project page under the [project name](understanding-radicle/glossary.md/#project-name), or in a seed node UI.
 
 [insert picture of projectID in seednode UI and in project page]
 
-Once found, copy the projectID into the Upstream search bar. If the project exists in your network of connected peers, it will appear. Following the project will replicate it to you local machine.
+Once found, copy the projectID into the Upstream search bar. Following the project will replicate it to your local machine once it's found in your network of connected peers.
 
 [insert picture of projectID & search bar]
 
 ### Cloning projects
 
-To clone a working copy of a followed project to your computer, you can use the [**Checkout**](understanding-radicle/glossary.md/#checkout) option within the Upstream client. This will clone the project to a specified location on your local machine and initialize a Git repository.
+To clone a working copy of a followed project to your computer, you can use the [**Checkout**](understanding-radicle/glossary.md/#checkout) option within the Upstream client. This will clone the project to a specified location on your local machine.
 
-Commiting and publishing changes to this working copy will create a new [remote](understanding-radicle/glossary.md/#remote) for that project that is uniquely linked to your peerID. You can view this remote in the peer switcher.
+Commiting and publishing changes to this working copy will create a new [remote](understanding-radicle/glossary.md/#remote) for that project which is uniquely linked to your peerID. You can view this remote in the peer switcher.
 
 [insert picture of peer switcher]
 
-Once a remote is created for a project, it will also add the project to your **My Projects** tab. 
+Once a remote is created for a project, it will also move the project to your **My Projects** tab.
```

Comment:
I think it's just called "Projects" in the UI, right?
```suggestion
Once a remote is created for a project, it will also move the project to your **Projects** tab.
```